operations/finance.md

```diff
@@ -664,6 +671,29 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 | `DisplayCid` | boolean | required | Display CID number on bill, only applicable for `Invoice`. |
 | `Rebated` | boolean | required | Bill has been rebated. |
 
+#### Bill assignee data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | string [Bill assignee data discriminator](#bill-assignee-data-discriminator) | required | Determines type of value. |
+| `Value` | object | required | Structure of object depends on [Bill assignee data discriminator](#bill-assignee-data-discriminator). |
+
+#### Bill assignee data discriminator
+
+* `BillCustomerData` - Assignee data specific to a customer.
+* `BillCompanyData` - Assignee data specific to a company.
+
+#### Bill customer data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+
+#### Bill company data
+
+| Property | Type |  | Description |
```

Comment:
Instead of empty table, we use juts "Empty object." I think, check some API responses that do not return anything and make the documentation consistent.